app/layout.tsx

```diff
@@ -81,26 +81,24 @@ export default function RootLayout({
                 id="mobile-header"
                 className={cn(
                   "md:hidden", // Mobile only
-                  "sticky top-0 z-10 grid w-full grid-cols-3 items-center py-4",
-                  "-mx-4 sm:-mx-4 md:-mx-8 xl:-mx-16", // Extend the background to the edges
-                  "px-4 sm:px-4 md:px-8 xl:px-16" // Padding to accommodate for margins
+                  "sticky top-0 z-10 flex w-full items-center justify-between p-6"
```

Comment:
Not sure if it is just me but the header doesn't stick

![image](https://github.com/user-attachments/assets/47b532b5-5151-4efc-ba6c-af74980a9a47)